examples/stable/mysql-csi-snapshot/README.md

```diff
@@ -0,0 +1,267 @@
+
+```bash
+$ mysql --user=root --password=asd#45@mysqlEXAMPLE
+
+mysql> CREATE DATABASE test;
+Query OK, 1 row affected (0.00 sec)
+
+mysql> USE test;
+Database changed
+
+# Create "employees" table
+mysql> CREATE TABLE employees (name VARCHAR(20), manager VARCHAR(20), address VARCHAR(200), sex CHAR(1), birth DATE, department VARCHAR(20));
+Query OK, 0 rows affected (0.02 sec)
+
+# Insert row to the table
+mysql> INSERT INTO employees VALUES ('Max','Tom','Naperville, Illinois, Chicago','m','1999-03-30','Engineering');
+Query OK, 1 row affected (0.01 sec)
+
+# View data in "employees" table
+mysql> SELECT * FROM employees;
++------+---------+-------------------------------+------+------------+-------------+
+| name | manager | address                       | sex  | birth      | department  |
++------+---------+-------------------------------+------+------------+-------------+
+| Max  | Tom     | Naperville, Illinois, Chicago | m    | 1999-03-30 | Engineering |
++------+---------+-------------------------------+------+------------+-------------+
+1 row in set (0.00 sec)
+```
+
+# Backup MySQL DB
+
+## Create Blueprint
+
+Create Blueprint in the same namespace as the Kanister controller.
+
+> **Note**: Please use appropriate snapshotClass and storageClass in the ./mysql-csi-snapshot-bp.yaml file. You can correct the blueprint using kubectl patch or kubectl edit commands.
+
+```bash
+$ kubectl create -f ./mysql-csi-snapshot-bp.yaml -n kanister
+```
+
+## Backup the application data
+
+Take a backup of the MySQL data using the backup ActionSet from above blueprint. Create an ActionSet in the same namespace as the controller. An easy way to do this is by using ``kanctl``.
+
+```bash
+# Create Actionset
+# Please make sure the value of blueprint matches with the name of blueprint that we created previously
+$ kanctl create actionset --action backup --namespace kanister --blueprint mysql-csi-snapshot-bp --pvc mysql/$(kubectl get pvc -n mysql --selector=app.kubernetes.io/instance=mysql-release -o=jsonpath='{.items[0].metadata.name}')
+actionset backup-mlvcv created
+
+$ kubectl --namespace kanister get actionset
+NAME                         AGE
+backup-mlvcv                 112s
+
+# View the status of the actionset
+# Please make sure the name of the actionset here matches with name of the name of actionset that we have created already
+$ kubectl --namespace kanister describe actionset backup-mlvcv
+
+# Check the CSI VolumeSnapshot created
+$ kubectl -n mysql get volumesnapshot
+NAME                                  READYTOUSE   SOURCEPVC              SOURCESNAPSHOTCONTENT   RESTORESIZE   SNAPSHOTCLASS      SNAPSHOTCONTENT                                    CREATIONTIME   AGE
+data-mysql-release-0-snapshot-cd26z   true         data-mysql-release-0                           8Gi           do-block-storage   snapcontent-2e411d1d-0b5f-48d4-9b79-b19e824c2e38   3h40m          3h40m
```

Comment:
Mention that you are using DO cluster hence the snapshhotclass is `do-block-storage`